content/en/docs/features/models.md

```diff
@@ -24,5 +24,7 @@ meta-llama/Meta-Llama-3.1-70B-Instruct       | Q2_K, Q3_K_M, Q3_K_S, Q4_0 |  vLL
 meta-llama/Meta-Llama-3.1-8B-Instruct        | None                       |  vLLM
 meta-llama/Meta-Llama-3.1-8B-Instruct        | AWQ                        |  vLLM, Triton
 meta-llama/Meta-Llama-3.1-8B-Instruct        | Q4_0                       |  vLLM, Ollama
+nvidia/Llama-3.1-Nemotron-70B-Instruct       | Q2_K, Q3_K_M, Q3_K_S, Q4_0 |  vLLM
+
```

Comment:
not sure if the empty line is added intentionally

Reply:
ah not intentional. let me fix.